### PR TITLE
PLT-1035: Remove last data point in graphs on #statistics page

### PR DESCRIPTION
**Pull request by florianorben**
Also sets correct hours/minutes/seconds

### CODE REVIEW

**Comment by it33** [comment]
Thanks! Really useful!

**Comment by jwilander** [comment]
+1

**Comment by jwilander** [comment]
@coreyhulen please take a look

**Comment by jwilander** [comment]
Merger please resolve https://mattermost.atlassian.net/browse/PLT-1035

File: api/admin_test.go [hunk 1]
```diff
@@ -235,6 +235,10 @@ func TestGetPostCount(t *testing.T) {
 	post1 := &model.Post{ChannelId: channel1.Id, Message: "a" + model.NewId() + "a"}
 	post1 = Client.Must(Client.CreatePost(post1)).Data.(*model.Post)
 
+	// manually update creation time, since it's always set to 0 upon saving and we only retrieve posts < today
```
Comment (coreyhulen) [inline api/admin_test.go]: hah...the comment threw me.  Should be something like, manually update creation time to yesterday.

**Comment by coreyhulen** [comment]
+1

**Pull request merged**